optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/roster/RosterGenerator.java

```diff
@@ -503,9 +504,8 @@ private RosterState createRosterState(GeneratorType generatorType, Integer tenan
                 for (int i = 0; i < stateCount; i++) {
                     Employee employee = availableEmployeeList.remove(random.nextInt(availableEmployeeList.size()));
                     // TODO Can this and ShiftTemplate.createShiftOnDate() be simplified and be DST spring/fall compatible?
-                    EmployeeAvailability employeeAvailability = new EmployeeAvailability(tenantId, employee, date,
-                            OffsetTime.of(LocalTime.MIN, zoneRules.getOffset(date.atStartOfDay())),
-                            OffsetTime.of(LocalTime.MAX, zoneRules.getOffset(date.plusDays(1).atStartOfDay()))); // TODO set to 00:00 next day instead
+                    EmployeeAvailability employeeAvailability = new EmployeeAvailability(tenantId, employee, date.atTime(OffsetTime.of(LocalTime.MIN, ZoneOffset.UTC)),
+                            date.plusDays(1).atTime(OffsetTime.of(LocalTime.MIN, ZoneOffset.UTC)));
```

Comment:
Remove the TODO above it as this fixes it :)

Reply:
The TODO is already removed